zerver/tests/test_tutorial.py

```diff
@@ -15,7 +15,12 @@ def setUp(self) -> None:
         user = self.example_user("hamlet")
         welcome_bot = get_system_bot(settings.WELCOME_BOT, user.realm_id)
         content = "Shortened welcome message."
-        internal_send_private_message(welcome_bot, user, content)
+        internal_send_private_message(
+            welcome_bot,
+            user,
+            content,
+            True,
```

Comment:
In addition to making this call via keyword argument ... we should explain why this parameter is set, and why it's important (will the tests fail if we don't have it here, or will it just be a bad simulation?).